frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -1,20 +1,40 @@
 import * as React from 'react';
+import { getResource } from 'kubevirt-web-ui-components';
 
 import { navFactory } from '@console/internal/components/utils';
 
 import { ResourceEventStream } from '@console/internal/components/events';
 import { DetailsPage } from '@console/internal/components/factory';
 import { K8sResourceKindReference } from '@console/internal/module/k8s';
 
+import { PodModel } from '@console/internal/models';
 import { VMDetailsFirehose } from './vm-details';
 import { VMDisksFirehose } from '../vm-disks';
 import { VMNics } from '../vm-nics';
 import { VMConsoleFirehose } from './vm-console';
 
+import { VirtualMachineInstanceMigrationModel, VirtualMachineInstanceModel } from '../../models';
+import { menuActionsCreator } from './menu-actions';
+
 // import { VmEvents } from './vm-events';
-// import { menuActions } from './menu-actions';
 
 export const VirtualMachinesDetailsPage = (props: VirtualMachinesDetailsPageProps) => {
```

Comment:
Why aren't we using e.g. `React.FC` type here?

Also, using the props destructuring form is much cleaner:

```ts
const Foo: React.FC<FooProps> = ({ name, namespace }) => {};
```

If you need direct access to props, simply use an alias when destructuring:

```ts
const Foo: React.FC<FooProps> = (props: { name, namespace }) => {};
```

Reply:
Fixed, it was not updated.

But the props still have to be there because they are passed down to the `DetailsPage`